packages/cli/package.json

```diff
@@ -60,7 +60,8 @@
     "@types/fs-extra": "^8.0.0",
     "@types/node": "^10",
     "@types/web3": "^1.0.18",
-    "globby": "^8"
+    "globby": "^8",
+    "typescript": "^3.5.3"
```

Comment:
no need. we are already using root's typescript definition. It's better to remove this

Reply:
[oclif uses this](https://github.com/celo-org/dev-cli/blob/79b7f298776580bbe70b4211b94caaf962228629/src/commands/readme.ts#L194) to detect if it should route to ts or js (really dumb)

I figured better to tie into this functionality then make more changes to oclif